.github/workflows/test-sql-adapter.yml

```diff
@@ -5,59 +5,62 @@
 
```

Comment:
Can you help me understand why this file changed? Perhaps it should be reverted. I know we'll need to update the Poetry install in the future, but perhaps that should be in a separate ticket.

I have now created a new ticket for updating _all_ of the Actions.